src/Datadog.Trace/Configuration/ConfigurationKeys.cs

```diff
@@ -338,5 +338,19 @@ internal static class Debug
             /// </summary>
             public const string ForceFallbackLookup = "DD_TRACE_DEBUG_LOOKUP_FALLBACK";
         }
+
+        internal static class FeatureFlags
+        {
+            /// <summary>
+            /// Feature Flag for whether to enable the updated ASP.NET resource names
+            /// </summary>
+            /// <seealso cref="TracerSettings.AspnetRouteTemplateResourceNamesEnabled"/>
+            public const string AspnetRouteTemplateResourceNamesEnabled = "DD_TRACE_ASPNET_ROUTE_TEMPLATE_RESOURCE_NAMES_ENABLED";
+
+            /// <summary>
+            /// Feature Flag for whether .NET Standard mode is enabled
```

Comment:
"What is .NET Standard mode?"
```suggestion
            /// Feature Flag: enables instrumenting calls to netstandard.dll (only applies to CallSite instrumentation)
```